### PR TITLE
Add leaderboard page showing top reviewers

### PR DESCRIPTION
**Pull request by MusikAnimal**

### CODE REVIEW

File: public_html/templates/base.html [hunk 1]
```diff
@@ -19,9 +19,10 @@
 </head>
 <body>
 <header class="app-header">
-	<img src="{{ siteUrl( 'images/copypatrol.png' ) }}" />
+	<a href="{{ siteUrl( '' ) }}"><img src="{{ siteUrl( 'images/copypatrol.png' ) }}" /></a>
```
Comment (kaldari) [inline public_html/templates/base.html]: Good idea.

File: src/Controllers/Leaderboard.php [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author MusikAnimal <musikanimal@wikimedia.org>
+ * @copyright © 2016 Niharika Kohli and contributors.
```
Comment (kaldari) [inline src/Controllers/Leaderboard.php]: At some point we need to change all these copyright notices to say "@copyright © 2016 Wikimedia Foundation", but that's not really relevant to this change.

File: public_html/templates/base.html [hunk 3]
```diff
@@ -73,43 +74,46 @@
...
-	</section>
+	{% block main %}
+		<section class="filters container-fluid">
+			<form>
+				<fieldset class="form-group filters-fieldset text-center">
+					{% for type, description in filterTypes %}
+						<span class="radio-inline">
+							<label>
+								<input type="radio" name="filter" value="{{ type }}"
+										{% if filter == type %} checked="checked"{% endif %}
```
Comment (kaldari) [inline public_html/templates/base.html]: Extra tab.

File: public_html/templates/leaderboard.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+{% block main %}
+  <section class="leaderboard container">
+    <a class="back-to-copypatrol" href="{{ siteUrl( '' ) }}">
+      <span class="glyphicon glyphicon-chevron-left"></span>
+      Back to CopyPatrol
+    </a>
+    <h1 class="text-center">Leaderboard</h1>
+
+    <div class="text-center clearfix">
+      <span class="leaderboard-column col-sm-4">
```
Comment (Niharika29) [inline public_html/templates/leaderboard.html]: Why are these spans and not div? Aren't spans usually meant for inline things only?

Comment (MusikAnimal) [inline public_html/templates/leaderboard.html]: They are, looks like the browser is converting them to `display: block` when they are side by side and have a `float` property, which is an interesting find. Semantically and visually they are still inline, however, and they actually become inline when the columns are broke on to separate lines – which is when I would expect it be `display:block`, not the other way around!

**Pull request merged**